hpx/runtime/threads/policies/static_queue_scheduler.hpp

```diff
@@ -72,7 +72,12 @@ namespace hpx { namespace threads { namespace policies
           : base_type(init, deferred_initialization)
         {}
 
-        virtual bool has_thread_stealing(std::size_t num_thread) const override
+        virtual bool has_work_stealing_core() const override
+        {
+            return false;
+        }
+
+        virtual bool has_work_stealing_numa() const override
```

Comment:
Shouldn't these just come from `scheduler_base` which checks if the flags are set? The static schedulers disable stealing initially but there's nothing stopping someone from enabling stealing at runtime. Or, if these two functions should always return false we should also throw if someone tries to enable stealing.

Reply:
The current design has a virtual call to check the flags, the schedulers that don't allow stealing, just reset the flags back to off when returning the flags, even if someone has manually set them on. I don't like it, but I just added a numa version to allow the set and get of the flags. 
Ideally we would just remove the schedulers that don't support these features, remove the virtual calls, not need to throw if someone tries to enable unsupported features. For now I wanted to simply get the new flag added.